drivers/secure_os/ws_api.cr

```diff
@@ -0,0 +1,204 @@
+require "placeos-driver"
+require "./ws_api_models"
+
+# docs: https://drive.google.com/file/d/1moo9NnFWukSf6fegxaZnSP5ShqSr_A03/view?usp=sharing
+
+class SecureOS::WsApi < PlaceOS::Driver
+  generic_name :SecureOS
+  descriptive_name "SecureOS WebSocket API"
+
+  uri_base "ws://secureos.server:8888/"
+  default_settings({
+    shared_host:   true,
+    rest_api_host: "http://172.16.1.120:8888",
+    basic_auth:    {
+      username: "srvc_acct",
+      password: "password!",
+    },
+    camera_types:  [] of String,
+    camera_states: [StateType::Attached, StateType::Armed, StateType::Alarmed],
+    camera_events: [] of String,
```

Comment:
camera_types specifies which camera types to subscribe to, if it is empty then all cameras will be subscribed to.
Same thing for camera_states and camera_events.
We could default states and events to the few examples provided in the docs?